test/libsolidity/semanticTests/array/function_array_cross_calls.sol

```diff
@@ -45,6 +45,6 @@ contract C {
 // compileViaYul: also
 // ----
 // test() -> 5, 6, 7
-// gas irOptimized: 354715
+// gas irOptimized: 354689
```

Comment:
Oh, wow, this test is actually hit now apparently :-)! I guess it's the ternary in ``h()``...